packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -34,29 +45,31 @@ BuildArch: noarch
 Requires: subscription-manager
 Obsoletes: %{name}-fact-plugin < %{version}-%{release}
 
-%if %{dnf_install} || 0%{?suse_version} >= 1500
+%if 0%{?fedora} > 18 || 0%{?rhel} > 6
+%if %{dnf_install}
 Requires: python3-subscription-manager-rhsm
 %else
 Requires: python-rhsm
 %endif
+Requires: crontabs
+%endif
 
-%if 0%{?suse_version}
-%if 0%{?suse_version} >= 1500
-BuildRequires: python3-devel
-Requires: python3-zypp-plugin
-%else
+%if 0%{?rhel} == 5
```

Comment:
I am not sure why we are introducing RHEL 5 back into the mix?

Reply:
I'm not sure either, maybe my spec file is really old, let me double check